_posts/2024-08-06-czi-eoss6-announcement.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Chan Zuckerberg Initiative considers scikit-learn an Essential Open Source Software"
+date: August 6, 2024
+categories:
+  - Funding
+tags:
+  - Open Source
+  - Funding
+  - Internship
+  - Diversity
+featured-image: sklearn_czi.png
+
+postauthors:
+  - name: Guillaume Lemaitre
+    website: https://github.com/glemaitre
+    image: guillaume-lemaitre.jpg
+  - name: Lucy Liu
+    website: https://github.com/lucyleeow
+    image: lucyliu.jpeg
+---
+<div>
+  <img src="/assets/images/posts_images/{{ page.featured-image }}" alt="">
+  {% include postauthor.html %}
+</div>
+
+We are delighted to announce that `scikit-learn` has been awarded a grant from
+the [Chan Zuckerberg Initiative (CZI)](https://chanzuckerberg.com/)'s [Essential Open
+Source Software for Science
+(EOSS)](https://chanzuckerberg.com/rfa/essential-open-source-software-for-science/)
+program. This grant is funded by [Wellcome Trust](https://wellcome.org/).
+As in previous rounds, this cycle supports open-source software projects that are
+essential to biomedical research. This is the third time that CZI EOSS supports
+`scikit-learn`.
+
+In this new grant, we will focus on improving the evaluation and inspection of
+predictive models.
+
+## Predictive models evaluation & inspection
+
+When building a machine learning pipeline for a specific research problem, two key
+aspects are closely connected: (i) design the pipeline and (ii) assess, analyze, and
+inspect it. Researchers strive to identify the optimal pipeline, maximizing specific
+evaluation metrics, while also seeking at explaining the validity and rationale behind
+the pipeline's predictions. This is the cornerstone of answering research
+questions. With this proposal we aim to improve and extend the available `scikit-learn`
+tools.
+
+`scikit-learn` provides building blocks for model evaluation and statistical analysis of
+results. Originally, this information was presented in a raw format and required
+expertise from scientists to create intuitive reports for outreach to peers and
+outsiders. Recently, the `scikit-learn` community developed displays to easily generate
+visual figures for communicating such results. However, these displays are still in
+their early development stages and do not leverage all available statistical analysis
+tools (i.e., cross-validation) from `scikit-learn`. Thus, we aim to expand these
+displays to use the right statistical tools and thus promote the adoption of best
+practices when reporting results. Additionally, we also intend to create new displays
+that are considered as "standards" during such analyses.
```

Comment:
Are you saying that the new displays will be 'standard' (i.e., commonly used/well-established in the field) or that using the new displays will become standard practice in people's analysis workflows?

Reply:
Let me reformulate it but I wanted to say that there is "standard" analysis and we are going to implement displays to depict visually those analysis.

Reply:
Perfect, it looks great!